src/kanalytics.js

```diff
@@ -210,7 +210,7 @@ export default class KAnalytics extends BasePlugin {
       clientVer: this.config.playerVersion,
       entryId: this.config.entryId,
       sessionId: this.config.sessionId,
-      uiConfId: this.config.uiConfId || 0,
+      uiConfId: this.config.uiConfId || -1,
```

Comment:
@odedhutzler make sure we want this change. as in v2 we send 0 for non uiconf.

Reply:
Solid point, so we stay with 0 for consistency